### ISSUE TITLE
NTR: Sepiapterin reductase low in fibroblasts (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8255]
New term request Sepiapterin reductase low in fibroblasts (segal_080212160001):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000563
- label: Reduced sepiapterin reductase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:22522443
- def: Acitivity of sepiapterin reductase (7,8-dihydrobiopterin:NADP+ oxidoreductase; EC 1.1.1.153) below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8255
- synonyms:
